tools/provision.py

```diff
@@ -851,10 +853,19 @@ def process_args():
     )
 
     # Use defaults from aws config, but allow user to override
-    parser.add_argument("--aws-profile", type=str, default="default")
-    parser.add_argument("--aws-region", type=str)
-    parser.add_argument("--aws-access-key-id", type=str)
-    parser.add_argument("--aws-access-key-secret", type=str)
+    parser.add_argument("--aws-profile", type=str, default=os.getenv("AWS_PROFILE"))
+    parser.add_argument(
+        "--aws-region", type=str, default=os.getenv("AWS_DEFAULT_REGION")
+    )
+    parser.add_argument(
+        "--aws-access-key-id", type=str, default=os.getenv("AWS_ACCESS_KEY_ID")
+    )
+    parser.add_argument(
+        "--aws-secret-access-key", type=str, default=os.getenv("AWS_SECRET_ACCESS_KEY")
+    )
+    parser.add_argument(
+        "--aws-session-token", type=str, default=os.getenv("AWS_SESSION_TOKEN")
```

Comment:
Do we need to update documentation for this new option here? https://github.com/FreeRTOS/iot-reference-stm32u5/blob/update_provision_script/Getting_Started_Guide.md